### DISCUSSION TITLE
Need help in running sunbird-lms-service and sunbird-user-service as getting error while running maven command mvn play2:run.

### DISCUSSION THREAD

**Discussion by Abhilash-Kadu** [Sunbird-Lern/Community#146] [Installation] [answered]
- I completed the steps as mentioned in https://github.com/Sunbird-Lern/userorg-service/blob/master/README.md
- The steps till **mvn clean install -DskipTests** executes succesfully.
- But cannot proceed further **Setp:4 Run the netty server using the following maven command in the path <project-base-path>/sunbird-lms-service/controller using command mvn play2:run**
- I have tried it using Java-8 and Java-11.
- I am attaching the screenshot for reference.
![Screenshot from 2023-07-31 11-54-37](https://github.com/Sunbird-Lern/Community/assets/126186600/34f54554-6591-486a-8222-fd08e6ac7538)

**Comment by BharathwajShankar** [comment, marked as answer]
@Abhilash-Kadu can we connect regarding this in a meeting and solve this?

**Reply by Abhilash-Kadu** [reply]
Yes sir it will be very good, just tell me when it is convenient for you.

**Reply by BharathwajShankar** [reply]
can we connect by 12:30?
please share the bridge to join here, I will join by the time.

**Reply by Abhilash-Kadu** [reply]
Ok sir. Google meet will be ok?

**Reply by BharathwajShankar** [reply]
Yes

**Reply by Abhilash-Kadu** [reply]
https://meet.google.com/qaa-zaaj-iks?authuser=0

**Comment by Abhilash-Kadu** [comment]
Solved thank you.

**Reply by BharathwajShankar** [reply]
From the call, it was identified that @Abhilash-Kadu was using Java 17 version because of which the above error occurred. We downgraded it to Java 11 and it was working as desired. hence it is marked as resolved.
